yarn.lock

```diff
@@ -1606,25 +1606,6 @@
     minimatch "^3.0.4"
```

Comment:
I wouldn't expect any changes in the yarn.lock. Replace the yarn.lock, stage changes, run `yarn` again -> should have no changes.